<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sorts</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/images/window.png</normaloff>:/images/window.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
QSpinBox {
background-color: qlineargradient(spread:reflect, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(204, 204, 204, 255), stop:1 rgba(255, 255, 255, 255));
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color: black;
font: bold 10px;
min-width:4m;
padding: 6px;
}
QSpinBox:hover {
background-color: qlineargradient(spread:reflect, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(204, 204, 204, 255), stop:1 rgba(255, 255, 255, 255));
border-style: inset;
}

/* LIST WIDGET STyLE*/
/*
QListWidget {
background-color: qlineargradient(spread:reflect, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(204, 204, 204, 255), stop:1 rgba(255, 255, 255, 255));
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color: black;
font: bold 12px;
min-width:4m;
padding: 6px;
}
/*
QListWidget::item:hover {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #FAFBFE, stop: 1 #DCDEF1);
 }

 QListView::item:selected {
     border: 1px solid #6a6ea9;
 }*/
QLCDNumber {
   background-color: qlineargradient(spread:reflect, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(170, 170, 127, 255), stop:1 rgba(255, 255, 255, 255));

     border-style: outset;
     border-width: 1px;
     border-radius: 10px;
     border-color: black;
     font: bold 12px;
     min-width:4m;
     padding: 6px;
 }
QMainWindow{
	background-image: url(:/images/DBGT.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Timer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetQList">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>81</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="GenerarQList">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Genera Aleitoriamente los numeros&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: qlineargradient(spread:reflect, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(170, 170, 127, 255), stop:1 rgba(255, 255, 255, 255));

     border-style: outset;
     border-width: 1px;
     border-radius: 10px;
     border-color: black;
     font: bold 12px;
     min-width:4m;
     padding: 6px;
 }
 QPushButton:pressed {
     background-color: qlineargradient(spread:reflect, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(170, 170, 127, 255), stop:1 rgba(255, 255, 255, 255));

     border-style: inset;
 }</string>
    </property>
    <property name="text">
     <string>Generar</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxQL1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Elije la cantidad de numeros a Generar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ordenar_QLists">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Combina las dos listas en una tercera&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: qlineargradient(spread:reflect, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(170, 170, 127, 255), stop:1 rgba(255, 255, 255, 255));

     border-style: outset;
     border-width: 1px;
     border-radius: 10px;
     border-color: black;
     font: bold 12px;
     min-width:4m;
     padding: 6px;
 }
 QPushButton:pressed {
     background-color: qlineargradient(spread:reflect, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(170, 170, 127, 255), stop:1 rgba(255, 255, 255, 255));

     border-style: inset;
 }</string>
    </property>
    <property name="text">
     <string>Metodo</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetFinal">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>81</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>548</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Extras</string>
    </property>
    <addaction name="actionAnimar"/>
    <addaction name="actionDesanimar"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de_QLists_vs_STL_Lists"/>
   </widget>
   <addaction name="menuExtras"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAnimar">
   <property name="text">
    <string>Animar</string>
   </property>
  </action>
  <action name="actionDesanimar">
   <property name="text">
    <string>Desanimar</string>
   </property>
  </action>
  <action name="actionAcerca_de_QLists_vs_STL_Lists">
   <property name="text">
    <string>Acerca de Sorts</string>
   </property>
  </action>
  <action name="actionHeapSort">
   <property name="text">
    <string>HeapSort</string>
   </property>
  </action>
  <action name="actionMergeSort">
   <property name="text">
    <string>MergeSort</string>
   </property>
  </action>
  <action name="actionQuickSort">
   <property name="text">
    <string>QuickSort</string>
   </property>
  </action>
  <action name="actionInsertionSort">
   <property name="text">
    <string>InsertionSort</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
